src/commonComponents/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,37 @@
+// prop-types
+import PropTypes from "prop-types";
+
+//css
+import "./Checkbox.css";
+
+const Checkbox = function (props) {
+  const { className, isChecked, label, htmlFor } = props;
+
+  return (
+    <div className={`${className} checkbox`}>
+      <input
+        className="cursor-pointer"
+        type="checkbox"
+        id={htmlFor}
+        value={isChecked}
+      />
+      <label
+        htmlFor={htmlFor}
+        className="flex justify-content-center align-items-center"
+      >
+        {label}{" "}
+      </label>
+    </div>
+  );
+};
+
+Checkbox.defaultProps = {
```

Comment:
if htmlFor is a prop it should also go into propTypes object, also here I can see you are destructuring 4 props but you have only 2 mentioned in the Proptypes